velox/exec/fuzzer/AggregationFuzzerOptions.h

```diff
@@ -21,6 +21,9 @@
 
 namespace facebook::velox::exec::test {
 
+// The type of aggregation fuzzer test.
+enum FuzzerType { PrestoFuzzerTest = 1, SparkFuzzerTest };
```

Comment:
Enums should use 'enum class' and their names should follow kFooBar naming convention.

However, this enum seems a bit too broad and doesn't tell the reader what's the difference between options. Consider using a more-specific flag that explicitly says whether group keys must be orderable or be just comparable.

Reply:
Thanks for the suggestion. Updated this PR to add 'orderableGroupKeys' in the AggregationFuzzerOptions.